tox.ini

```diff
@@ -13,16 +13,28 @@ commands =
     zope-testrunner --test-path=src []
 extras = test
 
-[testenv:coverage]
-basepython = python3
-commands =
-    coverage run -m zope.testrunner --test-path=src []
-    coverage report --fail-under=100
-deps =
-    coverage
-
 [testenv:flake8]
 basepython = python3
 skip_install = true
 deps = flake8
 commands = flake8 src setup.py
+
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
+
+[coverage:run]
```

Comment:
Question I asked on zopefoundation/meta#12: Does this configuration get picked up correctly when it's here and not in setup.cfg?